scapy/contrib/automotive/uds_scan.py

```diff
@@ -297,9 +299,54 @@ def post_execute(self, socket, state, global_configuration):
 class UDS_RDBIEnumerator(UDS_Enumerator):
     _description = "Readable data identifier per state"
 
+    @staticmethod
```

Comment:
This code should go to the connector and not be part of the enumerator, in my opinion

Reply:
In line `308` I'm filtering for the state. In the connector this is not possible afaik

Reply:
Otherwise I agree